### DISCUSSION TITLE
Available coordinates for export from Taglab

### DISCUSSION THREAD

**Discussion by nirivas** [cnr-isti-vclab/TagLab#61] [Q&A] [answered]
Hello there,

Is it possible to also export z coordinates if using a DEM? When I export I see the x and y centroids but not the z. 

Best,

Nico

**Comment by maxcorsini** [comment, marked as answer]
Hi nirivas,

You are right. z is not exported. This is a good suggestion for future release. 

Thank you.

Best,
Massimiliano

**Reply by nirivas** [reply]
Awesome! Thanks for the reply!
